snapfaas/src/configs.rs

```diff
@@ -91,7 +91,8 @@ impl ResourceManagerConfig {
                 .expect("Bad kernel path URL").path().to_string();
             // use `firerunner`'s default DEFAULT_KERNEL_CMDLINE
             // defined in firecracker/vmm/lib.rs
-            app.cmdline = None;
+            // TODO: Not None for debug
```

Comment:
I think I agree with this change, but why the TODO? What is there todo?

Reply:
I thought `app.cmdline` was set to None for a reason (comment above), so I commented it out only for debugging. TODO is just to remind me to change it back. But it is confusing so I'll remove it.